bin/node/src/chain_spec.rs

```diff
@@ -131,7 +131,7 @@ pub struct ChainParams {
     token_symbol: String,
 
     /// AccountIds of authorities forming the committee at the genesis (comma delimited)
-    #[clap(long, require_value_delimiter = true, parse(from_str = parse_account_id))]
+    #[clap(long, require_delimiter = true, value_delimiter = ',', parse(from_str = parse_account_id))]
```

Comment:
that's weird - we are using `clap = 3.2.12`, so `require_delimiter` should be deprecated in favor of `require_value_delimiter`... https://docs.rs/clap/3.2.12/clap/struct.Arg.html#method.require_delimiter

it seems that we should leave `require_value_delimiter`, but add:
 - [takes_value](https://docs.rs/clap/3.2.12/clap/builder/struct.Arg.html#method.takes_value)
 - [use_value_delimiter](https://docs.rs/clap/3.2.12/clap/builder/struct.Arg.html#method.use_value_delimiter)


(https://docs.rs/clap/3.2.12/clap/builder/struct.Arg.html#method.require_value_delimiter)

Reply:
done, although I kept the "," requirement - I don't see us using any other delimiter anyway